data/remnant/remnant jobs.txt

```diff
@@ -207,7 +207,7 @@ mission "Remnant: Bounty 5"
 		random < 5
 	npc kill
 		government "Korath"
-		personality heroic vindictive uninterested target marked
+		personality heroic vindictive uninterested target marked staying
```

Comment:
Does this line still require `uninterested` anymore? or is it being left for any specific purpose?
```suggestion
		personality heroic vindictive target marked staying
```